src/Interfaces.jl

```diff
@@ -1532,6 +1532,13 @@ function add_gids!(
   a
 end
 
+"""
+    add_gids!(a::PRange, gids::AbstractPData, neighbors_snd=nothing, neighbors_rcv=nothing; kwargs...)
+
+Add ghost-IDs to the `PRange` `a` based on the IDs in `gids`.
+
```

Comment:
Would be good with an explanation of what this does exactly, and in particular when it needs to be done, but I am not sure I can provide that. Thoughts?